runtime-sdk-macros/src/method_handler_derive.rs

```diff
@@ -116,11 +116,30 @@ pub fn derive_method_handler(impl_block: syn::ItemImpl) -> TokenStream {
         }
     };
 
+    let query_parameters_impl = {
+        quote! {
+            fn query_parameters<C: Context>(ctx: &mut C, _args: ()) -> Result<Parameters, Error> {
```

Comment:
Where does the `Parameters` type come from?

Reply:
Hm, so this currently works because every module has the `Parameters` type defined in the same scope. Using the `Parameters` method of the `Module` trait would indeed be better, I'll change it to that, so: `<Self as module::Module>::Parameters`